ddl/ddl_worker.go

```diff
@@ -212,26 +262,12 @@ func (d *ddl) finishDDLJob(t *meta.Meta, job *model.Job) (err error) {
 	return errors.Trace(err)
 }
 
-// getHistoryDDLJob gets a DDL job with job's ID form history queue.
-func (d *ddl) getHistoryDDLJob(id int64) (*model.Job, error) {
-	var job *model.Job
-
-	err := kv.RunInNewTxn(d.store, false, func(txn kv.Transaction) error {
-		t := meta.NewMeta(txn)
-		var err1 error
-		job, err1 = t.GetHistoryDDLJob(id)
-		return errors.Trace(err1)
-	})
-
-	return job, errors.Trace(err)
-}
-
 // handleDDLJobQueue handles DDL jobs in DDL Job queue.
 // shouldCleanJobs is used to determine whether to clean up the job in adding index queue.
-func (d *ddl) handleDDLJobQueue(shouldCleanJobs bool) error {
+func (w *worker) handleDDLJobQueue(d *ddlCtx, shouldCleanJobs bool) error {
```

Comment:
I see a lot of function of worker accept ddlCtx as a parameter. Can we make it as a attribute of worker?